packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -180,6 +180,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
     private firstVersionCall = true;
     private _snapshotSequenceNumber: number | undefined;
+    private readonly defaultFirstSummaryCacheExpiryTimeout = 60 * 1000; // 60 seconds.
```

Comment:
nit: `defaultSummarizerCacheExpryTimeout`. Not sure what first summary signifies but it could be for both summarizer and non-summarizer clients.

Reply:
`defaultSummarizerCacheExpryTimeoutMs`